initial commit. will need a look-over. (#1074)
temporalio/documentation: docs/server/namespaces.md

new history before discarding replication tasks. Even though some progress could rollback during failovers, Temporal
provides the guarantee that Workflows won’t get stuck and will continue to make forward progress.

## Automatic Forwarding on Namespaces

> Temporal supports automatic forwarding of Start, Signal, and Query requests to the active Cluster. This feature must be enabled through a dynamic conflict flag for the given Namespace.
> 
> Once enabled, these Tasks will be sent to the Parent Task Queue partition that matches that namespace, if it exists.

Reviewer: ```suggestion
Temporal supports automatic forwarding of Start, Signal, and Query requests to the active Cluster.
This feature must be enabled through a dynamic conflict flag for the given Namespace.

Once enabled, Tasks are sent to the Parent Task Queue partition that matches that Namespace, if it exists.
```
